lib/driver.js

```diff
@@ -423,16 +442,17 @@ class YouiEngineDriver extends BaseDriver {
         errno = 'EOK';
       }
 
-      if (!connected && retryCount === (MAX_RETRY_COUNT - 1)) {
-        logger.errorAndThrow('Failed to connect ' + MAX_RETRY_COUNT + ' times. Aborting.');
+      if (!connected && retryCount === MAX_RETRY_COUNT - 1) {
+        logger.errorAndThrow(
+          'Failed to connect ' + MAX_RETRY_COUNT + ' times. Aborting.'
+        );
```

Comment:
I assume removing the brackets doesn't functionally change anything, but it did make it more readable. Was this flagged by lint?

Reply:
Yeah linter did this automatically when I ran `npm run lint -- --fix`